YAXLib/YAXExceptions.cs

```diff
@@ -177,15 +226,17 @@ public override string Message
     /// Raised when the element value corresponding to some property is not present in the given XML file, when deserializing.
     /// This exception is raised during deserialization.
     /// </summary>
-    public class YAXElementValueMissingException : YAXException
+    public class YAXElementValueMissingException : YAXDeserializationException
     {
         #region Constructors
 
         /// <summary>
         /// Initializes a new instance of the <see cref="YAXAttributeMissingException"/> class.
         /// </summary>
         /// <param name="elementName">Name of the element.</param>
-        public YAXElementValueMissingException(string elementName)
+        /// <param name="lineInfo">IXmlLineInfo derived object, e.g. XElement, XAttribute containing line info</param>
+        public YAXElementValueMissingException(string elementName, IXmlLineInfo lineInfo)

```

Comment:
this is a breaking change. Please create an overload for backwards-compatibility.  (multiple times)